xsd/netex_framework/netex_reusableComponents/netex_deckPlan_version.xsd

```diff
@@ -291,14 +289,14 @@
 			<xsd:element ref="ClassOfUseRef" minOccurs="0"/>
 			<xsd:element name="FareClass" type="FareClassEnumeration" default="any" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Fare class for which capacity is specifyed. Default is any, i.e. capacity is for all classes.</xsd:documentation>
+					<xsd:documentation>Edit care class for which capacity is specifyed. Default is any, i.e. capacity is for all classes.</xsd:documentation>
```

Comment:
This whole file somehow undoes previous changes; i.e., the proposes changes make things worse.

Reply:
Can you give me a version that I should use?  The merging was hell, so I knew there would be problems. Are there more on the typo level or fundamental problems @trurlurl